examples/cloud_export_example_test.go

```diff
@@ -1,85 +1,253 @@
 //go:build examples
 // +build examples
 
-//nolint:testpackage,forbidigo
+//nolint:forbidigo,goconst,testpackage
 package examples
 
 import (
 	"context"
 	"fmt"
 	"testing"
 
-	cloudexportpb "github.com/kentik/api-schema-public/gen/go/kentik/cloud_export/v202101beta1"
+	"github.com/AlekSi/pointer"
+	"github.com/kentik/community_sdk_golang/kentikapi/models"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestDemonstrateCloudExportAPI(t *testing.T) {
+func TestDemonstrateCloudExportAPIWithAWS(t *testing.T) {
 	t.Parallel()
-	err := demonstrateCloudExportAPI()
+	err := demonstrateCloudExportAPIWithAWS()
 	assert.NoError(t, err)
 }
 
-func demonstrateCloudExportAPI() error {
+func TestDemonstrateCloudExportAPIWithAzure(t *testing.T) {
+	t.Parallel()
```

Comment:
my observation is this parallel execution makes analysis of the examples output harder when running all the examples

Reply:
Yes, that's why it is disabled in instructions in README (`-parallel 1`)

Reply:
For me parallel examples are useful

Reply:
OK